src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -92,11 +92,12 @@ limitations under the License.
     <field name="SupportsNames" mask="0x02"/>
   </bitmap>
 
-  <bitmap name="TemperatureSetpointHoldPolicyBitmap" type="bitmap8">
+  <enum name="AtomicRequestTypeEnum" type="enum8">
     <cluster code="0x0201"/>
```

Comment:
This is going to need to be an actual global enum, right?  We're adding support for those right now, but we need an issue tracking switching this to that, so that we don't forget.

In the meantime, it might be good to put this in a (new) global-enums.xml, not here, to also help us not forget.

Reply:
OK, relocated to (new) global-enums.xml